app/operations/placements/change_status.rb

```diff
@@ -23,7 +21,19 @@ def call
       changed_to: new_status
     }
 
+    if (add_disqualify_reason = %w[qualified reserved].exclude?(new_status))
+      placement.status = "disqualified"
+      placement_changed_params[:properties] = { reason: new_status.humanize }
```

Comment:
Why do we need humanize here? Why can'tt we use the same title as was written?

Reply:
You mean, to store events properties like this: 
```ruby
{ reason: "team_fit" }
```
instead of
```ruby
{ reason: "Team fit }
```
?
We'll have to humanize it before displaying in activities anyway. Is there any benefit in it?

Reply:
status is qualified/reserved/disqualified, on properties it should be the selected disqualify reason title. The title will have already humanized value, i.e. users won't save team_fit to reason's title, it will be "Team fit"